### PR TITLE
Created Church Finder Page, Church Item, Church Data interface and Su…

### PR DESCRIPTION
**Pull request by jeffordway**
Give this a check and see if it works

### CODE REVIEW

File: src/pages/ChurchFinder.tsx
Comment (ballroomdru) [inline src/pages/ChurchFinder.tsx:1]: I can't seem to get the imports to work. I do have the other files in place.

File: src/context/churchContext.ts
Comment (ballroomdru) [inline src/context/churchContext.ts:1]: Is this supposed to be a .ts file instead of a .tsx file?

File: src/components/ChurchFinder/ChurchItem.tsx
Comment (ballroomdru) [inline src/components/ChurchFinder/ChurchItem.tsx:1]: The import statement doesn't seem to want to work for me.

File: src/context/churchContext.ts
Comment (jeffordway) [reply src/context/churchContext.ts:1]: Yes

File: src/pages/ChurchFinder.tsx
Comment (jeffordway) [reply src/pages/ChurchFinder.tsx:1]: I do not understand the problem you are having.

File: src/components/ChurchFinder/ChurchItem.tsx
Comment (jeffordway) [reply src/components/ChurchFinder/ChurchItem.tsx:1]: Not sure what the problem is, which specific import statement is not working?

Comment (ballroomdru) [reply src/components/ChurchFinder/ChurchItem.tsx:1]: import { Church } from "../../context/churchContext";

File: src/pages/ChurchFinder.tsx
Comment (ballroomdru) [reply src/pages/ChurchFinder.tsx:1]: import { Church } from "../../context/churchContext"; is not working for me. I copied the code and the location exactly I thought.

**Pull request merged**